extensions/ql-vscode/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## [UNRELEASED]
 
+- Update variant analysis view to display the length of the shortest path for path queries. #3659
```

Comment:
Minor suggestion to match the style of the rest of the changelog:

```suggestion
- Update variant analysis view to display the length of the shortest path for path queries. [#3659](https://github.com/github/vscode-codeql/pull/3659)
```

Reply:
Thanks! I've now also changed the ID to refer to the PR as the other entries in the changelog.
(I originally linked to the issue)